package.json

```diff
@@ -13,6 +13,7 @@
     "build": "npm run sass:build && react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "sass": "sass --watch src/styles/scss:src/styles/css",
```

Comment:
@dazyang did the line below on 17 not work for you? 
using node-sass means we fewer external dependencies ie that the dev has sass installed as a global command.
I dont mind either way - just want to know if the below is broken